README.md

```diff
@@ -1,55 +1,58 @@
-# Talkable Referral Program API Gem
+# Talkable Referral Marketing API Gem
 [![](https://ci.solanolabs.com:443/Talkable/talkable-ruby/badges/branches/master?badge_token=c2445aee31992aafe3d8fda62fcde2708f6254f6)](https://ci.solanolabs.com:443/Talkable/talkable-ruby/suites/484176)
 
-Talkable Ruby Gem to make your own referral program in Sinatra or Rails application
+Referral marketing is one of the most powerful strategies for ecommerce sales growth. [Talkable](www.talkable.com) provides a rich platform for referral marketing. You can integrate sophisticated referral marketing into your own ecommerce site using the Talkable Ruby gem for a Rails or Sinatra application.
```

Comment:
Please change `www.talkable.com` to `https://www.talkable.com`.
Currently this results in a link to file on GitHub instead of an external link:
https://github.com/talkable/talkable-ruby/blob/master/www.talkable.com

You can see the preview of README.md here:
https://github.com/DanielKehoe/talkable-ruby/blob/master/README.md